.pre-commit-config.yaml

```diff
@@ -25,4 +25,5 @@ repos:
     hooks:
     -   id: mypy
         args: [--ignore-missing-imports, --py2]
+        files: source/jormungandr/jormungandr
```

Comment:
Do we have a Ticket to keep track of running Mypy on the rest of the code base ?

Reply:
We didn't, but now it's here: https://jira.kisio.org/browse/NAVP-1330